src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java

```diff
@@ -67,11 +67,12 @@ public void readAndProcess() {
                 logMap.put("Deserialised message id", deserializedMsg.getMessageId());
                 logMap.put("Thread Name", Thread.currentThread().getName());
                 logMap.put("Message Consumer ID", id);
-                logger.info(String.format("%s - Message deserialised successfully", id), logMap);
+                var messageId = deserializedMsg.getMessageId();
+                logger.infoContext(messageId, String.format("%s - Message deserialised successfully", id), logMap);
 
-                messageProcessorService.processMessage(deserializedMsg);
+                messageProcessorService.processMessage(id, deserializedMsg);
 
-                logger.info(String.format("%s - Message offset %s processed, committing offset", id, msg.getOffset()));
+                logger.infoContext(messageId, String.format("%s - Message offset %s processed, committing offset", id, msg.getOffset()));
```

Comment:
I know its not strictly to do with this pr, but it's a logging change so makes sense to do it here.

Could you add to this message, and the Message Offset Retrieved message, the Partition as well as the offset?

Something like "Message Offset: %s, Partition: %s retrieved"
As in staging and live offset alone isn't useful information.

Reply:
Updated this to include a partition.